src/cpp/session/modules/SessionProfiler.R

```diff
@@ -13,4 +13,54 @@
 #
 #
 
+.rs.addJsonRpcHandler("start_profiling", function(profilerOptions)
+{
+   tryCatch({
+      tempPath <- .Call("rs_profilesPath")
+      if (!dir.exists(tempPath)) {
+         dir.create(tempPath, recursive = TRUE)
+      }
+      
+   	fileName <- tempfile(fileext = ".rprof", tmpdir = tempPath)
 
+      Rprof(filename = fileName, line.profiling = TRUE)
```

Comment:
What happens if someone tries to start `Rprof()` when a separate `Rprof()` instance is already running? (e.g. if the user had manually executed `Rprof()` themselves at the console)